tasks/main.yml

```diff
@@ -44,7 +52,10 @@
     lc_ctype: "{{ postgres_locale }}"
     state: present
     template: template0
+  vars:
```

Comment:
This is an implementation detail at a higher level, no tasks should know if they are or aren’t operating in this mode...

Reply:
This task will fail without ssh pipelining being set to true. This is due to an underprivileged user trying to set permissions on the ansible tmp file. I've had issues with this even with our servers on ec2 and by default include it now in these create database/create user tasks. I'll remove it but SO is littered with this as a solution to this problem.